internal/generator/vector/normalize/dedot_labels.go

```diff
@@ -2,64 +2,38 @@ package normalize
 
 import (
 	. "github.com/openshift/cluster-logging-operator/internal/generator/framework"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/elements"
 	"github.com/openshift/cluster-logging-operator/internal/generator/vector/helpers"
+	"strings"
 )
 
-// Dedotting namespace labels
-// Replaces '.' and '/' with '_'
+const (
```

Comment:
nit: what about using go:embed feature and put this script to dedot.vrl file

Reply:
:shrug: We could, seems like an extra layer here